OpenIPSL/Examples/Solar/PSAT/testIrradiation.mo

```diff
@@ -2,11 +2,11 @@ within OpenIPSL.Examples.Solar.PSAT;
 model testIrradiation
   extends Modelica.Icons.Example;
   Modelica.Blocks.Sources.Constant const(k=25) annotation (Placement(transformation(
-        origin={-59.5887,-17.0499},
-        extent={{-3.3987,-3.3987},{3.3987,3.3987}})));
+        origin={0.0,0.0},
+        extent={{-59.54669086659841,-39.54669086659841},{-40.45330913340159,-20.45330913340159}},rotation = 0.0)));
   Modelica.Blocks.Sources.Constant const1(k=1000) annotation (Placement(transformation(
-        origin={-87.9874,22.9501},
-        extent={{-3.3987,-3.3987},{3.3987,3.3987}})));
+        origin={0.0,0.0},
+        extent={{-100.04368118521606,39.95631881478396},{-79.95631881478394,60.04368118521604}},rotation = 0.0)));
```

Comment:
I actually did this change with Dymola, and did not start from scratch with graphics (Which I maybe should have). I think the odd "origin" annotations are from OpenModelica. That said: I would be in favor of removing the examples, they don't look well thought out, several look rather like unit tests than examples.

Reply:
@lvanfretti: what is your take on this?

Reply:
@hubertus65 yes, they are unit test examples for the solar models.

@dietmarw we don't have other PV models in the library, and we need to have unit tests at least for the existing ones. They are pretty good w.r.t. to models used in standard tools. They were developed at a time when the tools did not incorporate their own (in this case this was implemented both in PowerFactory and Modelica) (see the attachment).

Instead of removing them, we should actually move them under ./Examples/UnitTests as planned for v2 release.

Since the problem is only in the annotations, it should be easy enough for one of us to do it, we just have to assign someone. @GiuseppeLaera can help.

Reply:
@GiuseppeLaera, I think the easiest way to fix this is to rebuild all PV examples in your favorite tool, but probably not in OpenModelica.

Reply:
Actually, you do not need to rebuild but simply move the blocks back and forth and redo the connections that you should have clean annotations again. That's at least my experience when using Dymola 2020 onwards.

Reply:
Just to add to this discussion, I have developed the PVD1 model according to PowerFactory.

Now I am working now on their own implementation of a PV model, which is I guess the one @dietmarw was referring to . I will also be adding some examples with those modes. A PR should follow soon.

Reply:
@tinrabuzin I don't know exactly what Dietmar was referring too, but the only solar models we have is the ones from PSAT and the one from Farhan that I mentioned above. So, it will be great if you can commit the "official" PowerFactory one and even better if you can provide the reference trajectories from PF. Please get in touch with @marcelofcastro for examples on how we are doing the regression testing, which you can also see here: 
[here](https://alsetlab.github.io/NYPAModelTransformation/)

Reply:
@lvanfretti Yes, sorry it was you mentioning the model not Dietmar.